src/OrchardCore.Themes/TheAdmin/wwwroot/Scripts/TheAdmin.js

```diff
@@ -7136,7 +7136,7 @@ $(function () {
   var magicToken = $("input[name=__RequestVerificationToken]").first();
 
   if (magicToken) {
-    $("body").on("click", "a[itemprop~='UnsafeUrl'], a[data-unsafe-url]", function () {
```

Comment:
Here there was a selector for a[data-unsafe-url] which has been removed because it's not used anywhere.

Reply:
Fixed